lightstep/resource_metric_condition.go

```diff
@@ -268,6 +261,30 @@ func getQuerySchema(includeSpansQuery bool) map[string]*schema.Schema {
 	return sma
 }
 
+func getFinalWindowOperationSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeList,
+		Optional: true,
+		MaxItems: 1,
+		MinItems: 1,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"operator": {
+					Type:         schema.TypeString,
+					Optional:     true,
+					ValidateFunc: validation.StringInSlice([]string{"min", "max", "avg"}, false),
```

Comment:
TODO: validate with Jake before merging?

Reply:
Approval has been given to go forward with 'avg' support in the final time window https://lightstep.slack.com/archives/C0242JADM5F/p1653597784428679

Reply:
Oh yay! I'm just behind on slack 👍